src/mm/mod.rs

```diff
@@ -297,7 +299,7 @@ pub(crate) fn print_information() {
 	arch::mm::virtualmem::print_information();
 }
 
-#[allow(dead_code)]
+/// Soft-deprecated in favor of `PhysAlloc`
```

Comment:
Maybe we can note that this deprecation does not apply for executable allocations.

Reply:
There are no such allocations right now, so I think we can remove those functions entirely in the future. I think the last such allocation was removed in https://github.com/hermit-os/kernel/commit/544c6e5b44fcfc42fb890737fb1bd4b091722a19#diff-499078c6ea49b14787bc425331ad2b552b919d74ac6add023d7c16e261a3af37R488.

Reply:
I have moved all non-device allocations (kernel stacks) to the heap.